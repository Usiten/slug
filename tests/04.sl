= 1 < 2;
= 1 + 1 * (100) < 200;
= 10 < 0;
a = 1;
b = 2;
= b < a;
= a < b;