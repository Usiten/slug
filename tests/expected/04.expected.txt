-- Stack Top --
>1
 0
 0
 1
 1
---------------
-- Var Data --
a: 1
b: 2
---------------
